Sources/EduIDExpansion/Controllers/Coordinators/PersonalInfoCoordinator.swift

```diff
@@ -0,0 +1,44 @@
+import UIKit
+
+class PersonalInfoCoordinator: CoordinatorType {
+    
+    var children: [CoordinatorType] = []
+    
+    var parent: CoordinatorType?
+    var navigationController: UINavigationController
+    
+    init(navigationController: UINavigationController) {
+        self.navigationController = navigationController
+        self.navigationController.navigationBar.tintColor = .backgroundColor
+    }
+    
+    func start() {
+        navigationController.navigationBar.topItem?.rightBarButtonItem = UIBarButtonItem(image: UIImage(systemName: "xmark"), style: .plain, target: self, action: #selector(dismissPersonalInfoFlow))
+        let logo = UIImageView(image: UIImage.eduIDLogo)
+        logo.width(92)
+        logo.height(36)
+        navigationController.navigationBar.topItem?.titleView = logo
+    }
+    
+    @objc
+    func dismissPersonalInfoFlow() {
+        navigationController.dismiss(animated: true)
+    }
+    
+    @objc
+    func drillDown() {
+        navigationController.pushViewController(EditPersonalInfoViewController(), animated: true)
+        navigationController.viewControllers.last?.navigationItem.hidesBackButton = true
+        navigationController.viewControllers.last?.navigationItem.leftBarButtonItem = UIBarButtonItem(image: .arrowBack, style: .plain, target: self, action: #selector(goBack))
+        navigationController.navigationBar.tintColor = .backgroundColor
+        let logo = UIImageView(image: .eduIDLogo)
+        logo.width(92)
+        logo.height(36)
+        navigationController.viewControllers.last?.navigationItem.titleView = logo
+    }
+    
+    @objc
+    func goBack() {
+        navigationController.popViewController(animated: true)
```

Comment:
A coordinator should, via the delegate pattern, inform its parent when it has finished. When that happens the parent coordinator is responsible for removing the coordinator from its children and with that end the child coordinator's life.

Reply:
Lets revisit this when I implement the personal info screen further